java/core/src/main/java/co/worklytics/psoxy/gateway/impl/CommonRequestHandler.java

```diff
@@ -210,7 +210,7 @@ public HttpEventResponse handle(HttpEventRequest request) {
                             .withPseudonymImplementation(pseudonymImplementation.get()));
                     }
 
-                    proxyResponseContent = sanitizer.sanitize(targetUrl, responseContent);
+                    proxyResponseContent = sanitizer.sanitize("GET", targetUrl, responseContent);
```

Comment:
:yellow_circle:  Use the original http method:
```suggestion
                    proxyResponseContent = sanitizer.sanitize(request.getHttpMethod(), targetUrl, responseContent);
```

All the APIs we have work with GET, but Dropbox works with POST for all methods

Reply:
:face_palm